content/faculty/history.md

```diff
@@ -0,0 +1,106 @@
+title: История, которой гордимся
+slug: history
+category: faculty
+tags: [_nav, faculty]
+submenu_index: 1
+---
+
+###<center>НАША ИСТОРИЯ</center>
+
+На нынешней территории Белоруссии до революции не было высших учебных заведений. Только в 1918 г. были созданы Витебский и Могилевский пединституты и Витебский архитектурный институт (просуществовал до 1923 г.). В 1919 г. Наркомпрос принял постановление «Об организации 15-ти советских университетов», однако в результате интервенции осуществить намеченное не представилось возможным. 
```

Comment:
Don't make lines that long. It can be really hard to read those if an editor doesn't wrap lines. Try to fit each line to 80 characters or less.